showmethekey-gtk/smtk-keys-win.c

```diff
@@ -344,16 +353,6 @@ static void smtk_keys_win_constructed(GObject *object)
 	win->box = gtk_box_new(GTK_ORIENTATION_VERTICAL, 0);
 	adw_window_set_content(ADW_WINDOW(win), win->box);
 
-	// Allow user to choose position by drag this.
-	win->header_bar = adw_header_bar_new();
-	adw_header_bar_set_show_start_title_buttons(
-		ADW_HEADER_BAR(win->header_bar), false);
-	adw_header_bar_set_show_end_title_buttons(
-		ADW_HEADER_BAR(win->header_bar), false);
-	win->handle = adw_window_title_new(_("Clickable"), NULL);
-	adw_header_bar_set_title_widget(ADW_HEADER_BAR(win->header_bar),
-					win->handle);
-	gtk_box_append(GTK_BOX(win->box), win->header_bar);
 
```

Comment:
I personally have no opinion about layer shell if it don't break other things, but if you insist to make myself unable to move my window, I cannot accept that. I am a GNOME lover.

Reply:
No problem I can add it back but only when is a floating window because it doesn't work when it is a layer.
Sorry for removing that, i was kinda desperate to make this work properly on hyprland (make the window not tile and be automatically positioned at the bottom) so probably have somethings to fix.